modules/broker-coordination/src/main/java/org/wso2/broker/coordination/rdbms/CoordinationConfiguration.java

```diff
@@ -43,13 +45,13 @@ public void setRdbmsCoordinationConfig(RdbmsCoordinationConfiguration rdbmsCoord
      */
     public static class RdbmsCoordinationConfiguration {
 
-        private String nodeId;
+        private String nodeId = UUID.randomUUID().toString();
```

Comment:
Shall we use a value like "GENERATED" as the default and generate the randomUUID when initializing?